### ISSUE TITLE
have a parser step for all values

### ISSUE THREAD

**Issue by PineappleRind** [PineappleRind/latinstudy#3]
## currently:
- values are parsed only in certain cases, scattered throughout the codebase

## what would fix it:
would be nice to have
- a value parser in its own file
- - would return an object, maybe typed ParsedValue, that would be used globally

this issue depends on #4

**Comment by PineappleRind** [comment]
in progress
